packages/medusa/src/commands/seed.ts

```diff
@@ -131,6 +134,7 @@ const seed = async function ({ directory, migrate, seedFile }: SeedOptions) {
     const {
       store: seededStore,
       regions,
+      product_collections,
```

Comment:
todo: Default to empty array
```suggestion
      product_collections = [],
```

Reply:
Good point, done :)